packages/sources/alpine/src/config.ts

```diff
@@ -0,0 +1,12 @@
+import { Requester, util } from '@chainlink/ea-bootstrap'
+import { Config } from '@chainlink/types'
+
+export const NAME = 'ALCHEMY' // This should be filled in with a name corresponding to the data provider using UPPERCASE and _underscores_.
+
+export const makeConfig = (prefix?: string): Config => {
+  const config = Requester.getDefaultConfig(prefix)
+
+  config.apiKey = util.getRequiredEnv('ALCHEMY_API_KEY')
```

Comment:
Use RPC_URL instead. There are several examples of how to do this if you grep the repo for `util.getRequiredEnv('RPC_URL')`.